notebooks/04_model_select_and_optimize/azureml_hyperdrive_tuning_svd.ipynb

```diff
@@ -0,0 +1,731 @@
+    "2. Select **Create a resource** menu, search for **Machine Learning service workspace** select **Create** button.\n",
+    "3. In the **ML service workspace** pane, configure your workspace by entering the *workspace name* and *resource group* (or **create new** resource group if you don't have one already), and select **Create**. It can take a few moments to create the workspace.\n",
+    "  \n",
+    "#### Make a configuration file\n",
+    "To configure this notebook to communicate with your workspace, type in your Azure subscription id, the resource group name and workspace name to <subscription-id>, <resource-group>, <workspace-name> in the cell below. Alternatively, you can create a *./aml_config/config.json* file with the following contents:\n",
+    "```\n",
+    "{\n",
+    "    \"subscription_id\": \"<subscription-id>\",\n",
+    "    \"resource_group\": \"<resource-group>\",\n",
+    "    \"workspace_name\": \"<workspace-name>\"\n",
+    "}\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Found the config file in: C:\\Users\\anargyri\\aml_config\\config.json\n",
+      "AzureML workspace name:  anargyri\n"
+     ]
+    }
+   ],
+   "source": [
+    "# AzureML workspace info. Note, will look up \"aml_config\\config.json\" first, then fall back to use this\n",
+    "SUBSCRIPTION_ID = '<subscription-id>'\n",
+    "RESOURCE_GROUP  = '<resource-group>'\n",
+    "WORKSPACE_NAME  = '<workspace-name>'\n",
+    "\n",
+    "# Connect to a workspace\n",
+    "try:\n",
+    "    ws = aml.core.Workspace.from_config()\n",
+    "except aml.exceptions.UserErrorException:\n",
+    "    try:\n",
+    "        ws = aml.core.Workspace(\n",
+    "            subscription_id=SUBSCRIPTION_ID,\n",
+    "            resource_group=RESOURCE_GROUP,\n",
+    "            workspace_name=WORKSPACE_NAME\n",
+    "        )\n",
+    "        ws.write_config()\n",
+    "    except aml.exceptions.AuthenticationException:\n",
+    "        ws = None\n",
+    "\n",
+    "if ws is None:\n",
+    "    raise ValueError(\n",
+    "        \"\"\"Cannot access the AzureML workspace w/ the config info provided.\n",
+    "        Please check if you entered the correct id, group name and workspace name\"\"\"\n",
+    "    )\n",
+    "else:\n",
+    "    print(\"AzureML workspace name: \", ws.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the following cells, we\n",
+    "1. Create a *remote compute target* (cpu_cluster) if it does not exist already,\n",
+    "2. Mount a *data store* and upload the training set, and\n",
+    "3. Run a hyperparameter tuning experiment."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a Remote Compute Target\n",
```

Comment:
Is it better that this can be refered to the documentation of AzureML where it is introduced?

Reply:
There is already a link to https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets